mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDaoImpl.java

```diff
@@ -381,4 +425,37 @@ private org.folio.rest.jaxrs.model.ActionStatus mapNameToEntityActionStatus(Stri
   private JobLogEntryDto.SourceRecordType mapToEntityType(String entityType) {
     return entityType == null ? null : JobLogEntryDto.SourceRecordType.fromValue(entityType);
   }
+
+  private JobExecutionSummaryDto mapRowSetToJobExecutionSummaryDto(RowSet<Row> rowSet) {
+    JobExecutionSummaryDto jobExecutionSummaryDto = new JobExecutionSummaryDto();
+
+    rowSet.forEach(row -> jobExecutionSummaryDto
```

Comment:
Hi @RuslanLavrov as I understand, we'll only have one row, isn't it better to use rowSet.iterator().next() and make it as "return new JobExecutionSummaryDto.with.....".
It will make the code more clear.